SQF/dayz_code/stringtable.xml

```diff
@@ -16729,7 +16829,7 @@
 			<Russian>Вы не можете снять детали в безопасной зоне</Russian>
 			<French>Vous ne pouvez pas récupérer des pièces dans une zone sécurisée</French>
 			<Czech>Nemůžete zachránit díly v bezpečné zóně</Czech>
-			<German>Du kannst in einer Safezone keine Teile demontieren</German>
+			<German>Du kannst in einer Safe Zone keine Teile demontieren</German>
```

Comment:
I would leave that because it is an english word that we uses and all safe zone scripts spelling it like safe zone not safezone thats why I changed it in Safe Zone. Translating is not a good idea here I think.

Reply:
Agree